docs/guide_demo_app_contribution.md

```diff
@@ -0,0 +1,167 @@
+---
+id: guide_demo_app_contribution
+title: Demo App Contributor Guide
+sidebar_label: Demo App Contributor Guide
+---
+[Getting Started with Demo App](#demoappcontributorguide-gettingstartedwithdemoapp)
+
+[Folder Structure](#demoappcontributorguide-folderstructure)
+
+&nbsp;&nbsp;&nbsp;&nbsp;[Index](#demoappcontributorguide-index)
```

Comment:
Can you use markdown syntax for unordered lists here?
```md
[Folder Structure](#demoappcontributorguide-folderstructure)
- [Index](#demoappcontributorguide-index)
- [Includes](#demoappcontributorguide-includes)
```

Reply:
Actually, this whole table of contents isn't needed because it looks like docusaurus automatically creates the ToC on the right side based on the headers, see: https://deploy-preview-933--upbeat-shaw-3c738f.netlify.app/docs/guide_demo_app_contribution

Reply:
done